### PR TITLE
Fix many spelling errors

### LINKED ISSUE #3270: (Probably comprehensive) list of spelling errors
I was making an EIP (https://github.com/ethereum/EIPs/pull/4835) and including OpenZeppelin as a submodule in the assets (because I was using it in my reference impls), when the CI failed due to the following spelling errors:

```./assets/eip-4834/OpenZeppelin/package-lock.json:9342: vEw ==> view
./assets/eip-4834/OpenZeppelin/package-lock.json:16026: vEw ==> view
./assets/eip-4834/OpenZeppelin/package-lock.json:16641: vEw ==> view
./assets/eip-4834/OpenZeppelin/package-lock.json:26020: vEw ==> view
./assets/eip-4834/OpenZeppelin/package-lock.json:31325: vEw ==> view
./assets/eip-4834/OpenZeppelin/package-lock.json:31825: vEw ==> view
./assets/eip-4834/OpenZeppelin/CHANGELOG.md:459: libray ==> library
./assets/eip-4834/OpenZeppelin/certora/applyHarness.patch:74: consistant ==> consistent
./assets/eip-4834/OpenZeppelin/certora/specs/GovernorBase.spec:176: completness ==> completeness
./assets/eip-4834/OpenZeppelin/certora/specs/GovernorBase.spec:177: chages ==> changes, charges
./assets/eip-4834/OpenZeppelin/certora/specs/GovernorBase.spec:180: seperately ==> separately
./assets/eip-4834/OpenZeppelin/certora/specs/GovernorCountingSimple.spec:131: completness ==> completeness
./assets/eip-4834/OpenZeppelin/certora/specs/GovernorCountingSimple.spec:132: chages ==> changes, charges
./assets/eip-4834/OpenZeppelin/certora/specs/GovernorCountingSimple.spec:135: seperately ==> separately
./assets/eip-4834/OpenZeppelin/certora/specs/GovernorCountingSimple.spec:208: priveleged ==> privileged
./assets/eip-4834/OpenZeppelin/certora/specs/GovernorCountingSimple.spec:220: priveleged ==> privileged
./assets/eip-4834/OpenZeppelin/contracts/interfaces/IERC2981.sol:19: payed ==> paid
./assets/eip-4834/OpenZeppelin/contracts/utils/structs/DoubleEndedQueue.sol:27: could't ==> couldn't
./assets/eip-4834/OpenZeppelin/contracts/utils/math/SafeMath.sol:31: substraction ==> subtraction
./assets/eip-4834/OpenZeppelin/contracts/proxy/Proxy.sol:48: overriden ==> overridden
./assets/eip-4834/OpenZeppelin/contracts/proxy/Proxy.sol:83: overriden ==> overridden
./assets/eip-4834/OpenZeppelin/contracts/governance/README.adoc:43: requiering ==> requiring
./assets/eip-4834/OpenZeppelin/contracts/governance/TimelockController.sol:188: datas ==> data
./assets/eip-4834/OpenZeppelin/contracts/governance/TimelockController.sol:192: datas ==> data
./assets/eip-4834/OpenZeppelin/contracts/governance/TimelockController.sol:229: datas ==> data
./assets/eip-4834/OpenZeppelin/contracts/governance/TimelockController.sol:235: datas ==> data
./assets/eip-4834/OpenZeppelin/contracts/governance/TimelockController.sol:237: datas ==> data
./assets/eip-4834/OpenZeppelin/contracts/governance/TimelockController.sol:240: datas ==> data
./assets/eip-4834/OpenZeppelin/contracts/governance/TimelockController.sol:304: datas ==> data
./assets/eip-4834/OpenZeppelin/contracts/governance/TimelockController.sol:309: datas ==> data
./assets/eip-4834/OpenZeppelin/contracts/governance/TimelockController.sol:311: datas ==> data
./assets/eip-4834/OpenZeppelin/contracts/governance/TimelockController.sol:314: datas ==> data
./assets/eip-4834/OpenZeppelin/contracts/governance/Governor.sol:49: consummed ==> consumed
./assets/eip-4834/OpenZeppelin/contracts/governance/Governor.sol:67: poping ==> popping, pooping
./assets/eip-4834/OpenZeppelin/contracts/governance/Governor.sol:127: accross ==> across
./assets/eip-4834/OpenZeppelin/contracts/governance/Governor.sol:232: overriden ==> overridden
./assets/eip-4834/OpenZeppelin/contracts/governance/Governor.sol:311: overriden ==> overridden
./assets/eip-4834/OpenZeppelin/contracts/governance/Governor.sol:328: trigerred ==> triggered
./assets/eip-4834/OpenZeppelin/contracts/governance/Governor.sol:347: trigerred ==> triggered
./assets/eip-4834/OpenZeppelin/contracts/governance/IGovernor.sol:161: snaphot ==> snapshot
./assets/eip-4834/OpenZeppelin/contracts/governance/IGovernor.sol:162: quroum ==> quorum
./assets/eip-4834/OpenZeppelin/contracts/governance/extensions/GovernorTimelockControl.sol:50: Overriden ==> Overridden
./assets/eip-4834/OpenZeppelin/contracts/governance/extensions/GovernorTimelockControl.sol:110: Overriden ==> Overridden
./assets/eip-4834/OpenZeppelin/contracts/governance/extensions/GovernorTimelockControl.sol:123: Overriden ==> Overridden
./assets/eip-4834/OpenZeppelin/contracts/governance/extensions/GovernorTimelockCompound.sol:108: Overriden ==> Overridden
./assets/eip-4834/OpenZeppelin/contracts/governance/extensions/GovernorTimelockCompound.sol:170: Overriden ==> Overridden
./assets/eip-4834/OpenZeppelin/contracts/governance/extensions/GovernorTimelockCompound.sol:188: Overriden ==> Overridden
./assets/eip-4834/OpenZeppelin/contracts/finance/VestingWallet.sol:123: amout ==> amount
./assets/eip-4834/OpenZeppelin/test/security/Pausable.test.js:70: process' ==> process's
./assets/eip-4834/OpenZeppelin/test/helpers/governance.js:109: otherwize ==> otherwise
./assets/eip-4834/OpenZeppelin/test/helpers/governance.js:115: otherwize ==> otherwise
./assets/eip-4834/OpenZeppelin/test/helpers/governance.js:168: queing ==> queueing
./assets/eip-4834/OpenZeppelin/test/utils/structs/DoubleEndedQueue.test.js:72: begining ==> beginning
./assets/eip-4834/OpenZeppelin/test/utils/cryptography/ECDSA.test.js:50: siganture ==> signature
./assets/eip-4834/OpenZeppelin/test/governance/TimelockController.test.js:28: datas ==> data
./assets/eip-4834/OpenZeppelin/test/governance/TimelockController.test.js:38: datas ==> data
./assets/eip-4834/OpenZeppelin/test/governance/TimelockController.test.js:42: datas ==> data
./assets/eip-4834/OpenZeppelin/test/governance/TimelockController.test.js:122: datas ==> data
./assets/eip-4834/OpenZeppelin/test/governance/TimelockController.test.js:166: overwritting ==> overwriting
./assets/eip-4834/OpenZeppelin/test/governance/TimelockController.test.js:349: datas ==> data
./assets/eip-4834/OpenZeppelin/test/governance/TimelockController.test.js:361: datas ==> data
./assets/eip-4834/OpenZeppelin/test/governance/TimelockController.test.js:373: overwritting ==> overwriting
./assets/eip-4834/OpenZeppelin/test/governance/TimelockController.test.js:377: datas ==> data
./assets/eip-4834/OpenZeppelin/test/governance/TimelockController.test.js:388: datas ==> data
./assets/eip-4834/OpenZeppelin/test/governance/TimelockController.test.js:403: datas ==> data
./assets/eip-4834/OpenZeppelin/test/governance/TimelockController.test.js:433: datas ==> data
./assets/eip-4834/OpenZeppelin/test/governance/TimelockController.test.js:448: datas ==> data
./assets/eip-4834/OpenZeppelin/test/governance/TimelockController.test.js:475: datas ==> data
./assets/eip-4834/OpenZeppelin/test/governance/TimelockController.test.js:489: datas ==> data
./assets/eip-4834/OpenZeppelin/test/governance/TimelockController.test.js:502: datas ==> data
./assets/eip-4834/OpenZeppelin/test/governance/TimelockController.test.js:519: datas ==> data
./assets/eip-4834/OpenZeppelin/test/governance/TimelockController.test.js:538: datas ==> data
./assets/eip-4834/OpenZeppelin/test/governance/TimelockController.test.js:549: datas ==> data
./assets/eip-4834/OpenZeppelin/test/governance/TimelockController.test.js:559: datas ==> data
./assets/eip-4834/OpenZeppelin/test/governance/TimelockController.test.js:573: datas ==> data
./assets/eip-4834/OpenZeppelin/test/governance/TimelockController.test.js:587: datas ==> data
./assets/eip-4834/OpenZeppelin/test/governance/TimelockController.test.js:636: datas ==> data
./assets/eip-4834/OpenZeppelin/test/governance/TimelockController.test.js:647: datas ==> data
./assets/eip-4834/OpenZeppelin/test/governance/compatibility/GovernorCompatibilityBravo.test.js:218: doesnt ==> doesn't, does not
./assets/eip-4834/OpenZeppelin/audit/2017-03.md:136: modifers ==> modifiers
```

Some of these are likely false positives, but others are likely true spelling errors (e.g. `libray`, `datas`).

### PR DESCRIPTION
**Pull request by Pandapip1**
Fixes #3270

- Fixes lots of spelling mistakes

#### PR Checklist

- [x] Tests
- [x] Documentation
- [x] Changelog entry

(None of the above apply)

### CODE REVIEW

File: contracts/utils/math/SafeMath.sol [hunk 1]
```diff
@@ -28,7 +28,7 @@ library SafeMath {
     }
 
     /**
-     * @dev Returns the substraction of two unsigned integers, with an overflow flag.
+     * @dev Returns the subtraction of two unsigned integers, with an overflow flag.
```
Comment (Amxx) [inline contracts/utils/math/SafeMath.sol:31]: I don't think that change is correct

**Comment by Amxx** [comment]
Hello @Pandapip1 and thank you for submitting this PR.

While I (obviously) agree with most of the changes, I would like to discuss the use of "datas" in the TimelockController. The issue I have is that since the plural of data is data, there is no way to mark that this variable must contain multiple "data" entries (one per call).

We used this abusive form `datas` as a variable name to mark the fact should be an array of calls. In the Governor, we have a similar parameter called `calldatas`. Maybe we should use that.

@frangio can you find any better name for this field?

File: contracts/utils/math/SafeMath.sol (hunk 1 above)
Comment (Pandapip1) [reply contracts/utils/math/SafeMath.sol:31]: image

**Comment by Pandapip1** [comment]
Maybe `alldata`?

File: contracts/utils/math/SafeMath.sol (hunk 1 above)
Comment (Amxx) [reply contracts/utils/math/SafeMath.sol:31]: And Wikipedia also agrees...

Note that Google translate caused the confusion in the first place as it says "substraction" when translating "soustraction" from the French.

**Comment by Amxx** [comment]
Maybe "payloads" would be a good option. What do you think ?

**Comment by Pandapip1** [comment]
Changed `data` to `payloads`

**Pull request merged**